### DISCUSSION TITLE
Does anyone bind Android native jar or aar?

### DISCUSSION THREAD

**Discussion by bbhxwl** [AvaloniaUI/Avalonia#16311] [Q&A]
Does anyone bind Android native jar or aar?

**Comment by maxkatz6** [comment]
Avalonia doesn't manage these. .NET SDK does.
https://learn.microsoft.com/en-us/dotnet/android/building-apps/build-items#androidlibrary
See https://learn.microsoft.com/en-us/dotnet/maui/migration/android-binding-projects?view=net-maui-8.0
